scripts/idl/matter_idl_parser.py

```diff
@@ -252,7 +253,7 @@ def event_with_access(self, args):
         return init_args
 
     def event(self, args):
-        return Event(priority=args[0], code=args[2], fields=args[3:], **args[1])
+        return Event(attributes=args[0], priority=args[1], code=args[3], fields=args[4:], **args[2])
```

Comment:
Suggest qualities

Reply:
Done. This was a very large change, but for now trusting that unit tests caught any issues.